forte/data/base_pack.py

```diff
@@ -193,10 +194,27 @@ def _deserialize(
 
         if serialize_method == "jsonpickle":
             with _open(data_source, mode="rt") as f:  # type: ignore
-                pack = cls.from_string(f.read())
+                try:
+                    pack = cls.from_string(f.read())
+                except AttributeError as e:
+                    raise AttributeError(
+                        "There was an unknown type or attribute during"
+                        "deserialization and broke the process."
+                        "It might be caused by adding"
+                        "new classes or new attributes to the ontology."
+                    ) from e
+
         else:
             with _open(data_source, mode="rb") as f:  # type: ignore
-                pack = pickle.load(f)
+                try:
+                    pack = pickle.load(f)
+                except AttributeError as e:
+                    raise AttributeError(
+                        "There was an unknown type or attribute during"
+                        "deserialization and broke the process."
+                        "It might be caused by adding"
+                        "new classes or new attributes to the ontology."
```

Comment:
Can we have a test case to see if this exception will be raised?

Btw, here we have a similar problem: https://github.com/asyml/forte/blob/master/forte/data/data_pack.py#L134, and we are able to identify which entry class cause the problem, can we do something similar?

Reply:
Test case?